packages/mds-audit-api/tests/api.spec.ts

```diff
@@ -115,7 +116,11 @@ describe('Testing API', () => {
       propulsion_types: ['electric'],
       vehicle_type: 'scooter',
       recorded: AUDIT_START
-    }).then(() => {
+    }
+
+    Sinon.replace(IngestServiceClient, 'getDevice', Sinon.fake.resolves(device))
```

Comment:
Is this so we don't have to run the IngestServiceClient for this test suite?

Reply:
Yeup.